### PR TITLE
WIP: Experimental clang-format changes

### PR DESCRIPTION
**Pull request by hjmjohnson**
This is an experimental commit for applying clang-format to make the code consistent with ITK.

### CODE REVIEW

**Comment by SimonRit** [comment]
Thanks a lot, @LucasGandel was planning on doing something similar. Sorry, I hadn't noticed that your previous commits were not respecting ITK's clang-format style. I have a script, utilities/clang-format.bash, to apply .clang-format which I was running from within the ITK source directory. I guess it can be deleted if you copy the file?
I prefer no to touch the utilities directory (except ITKCudaCommon which will be made a remote module one day...), do you mind not touching those files? This is similar to the ThirdParty directory of ITK. Or have you also applied .clang-format to the ThirdParty directory?
include/rtkMechlemOneStepSpectralReconstructionFilter.h modifications should not be kept like this, it's harder to read the dot graph code (which is already quite unreadable). Can you remove clang-format automated change and split [line 91](https://github.com/SimonRit/RTK/pull/311/files#diff-1f0eed7147828fb25040b5d332e13fd9L91) in two like,e.g., [above](https://github.com/SimonRit/RTK/pull/311/files#diff-1f0eed7147828fb25040b5d332e13fd9L84-L85).
Thanks again for your efforts, it's very much appreciated.

File: include/rtkMechlemOneStepSpectralReconstructionFilter.h [hunk 1]
```diff
@@ -88,14 +88,14 @@ namespace rtk
...
- * Alphak [ label="", fixedsize="false", width=0, height=0, shape=none];
- * NextAlphak [ label="", fixedsize="false", width=0, height=0, shape=none];
+ * MultiplyGradientToBeBackProjected [ label="itk::MultiplyImageFilter" URL="\ref itk::MultiplyImageFilter"
+ * style=dashed]; AddGradients [ label="itk::AddImageFilter" URL="\ref itk::AddImageFilter"]; AddHessians [
+ * label="rtk::AddMatrixAndDiagonalImageFilter" URL="\ref rtk::AddMatrixAndDiagonalImageFilter"]; Newton [
+ * label="rtk::GetNewtonUpdateImageFilter" URL="\ref rtk::GetNewtonUpdateImageFilter"]; Nesterov [
+ * label="rtk::NesterovUpdateImageFilter" URL="\ref rtk::NesterovUpdateImageFilter"]; MultiplySupport [
+ * label="itk::MultiplyImageFilter" URL="\ref itk::MultiplyImageFilter" style=dashed]; Alphak [ label="",
+ * fixedsize="false", width=0, height=0, shape=none]; NextAlphak [ label="", fixedsize="false", width=0, height=0,
+ * shape=none];
```
Comment (SimonRit) [inline include/rtkMechlemOneStepSpectralReconstructionFilter.h:98]: If you adjust manually this change, to only move `style-dashed];` to the following line, this should the other lines unchanged which is preferable for readability. When this is done I can merge the PR, the rest looks good.

**Comment by SimonRit** [comment]
@hjmjohnson What is the status of this PR?

**Comment by hjmjohnson** [comment]
@SimonRit most of this was accepted in other PR's.   This PR can be closed.

**Pull request closed without merging**